content/en/community/hackathons/sessions/testing-unikraft/content/Work-Items_03.-Tutorial:-Testing-vfscore.md

```diff
@@ -69,6 +69,12 @@ UK_TESTCASE(vfscore_stat_testsuite, vfscore_test_newfile)
 /* Register the test suite */
 uk_testsuite_register(vfscore_stat_testsuite, NULL);
 ```
+After you select vfscore in `make menuconfig` a new option should appear in the `Library Configuration` menu to
+select `ramfs` filesystem:
```

Comment:
```suggestion
After you select vfscore in the configuration screen, a new option should appear in the `Library Configuration` menu to select a `ramfs` filesystem:
```
Please use one line per sentence and one sentence per line.

Reply:
Please use the 1st person pronouns (`we`) instead of 2nd person (`you`), to keep consistency in the whole file.